src/main/java/io/papermc/hangar/service/UserService.java

```diff
@@ -160,16 +162,16 @@ public UserData getUserData(UsersTable user) {
             throw new ResponseStatusException(HttpStatus.NOT_FOUND);
         }
         // TODO getUserData
-        boolean isOrga = false;
+        boolean isOrga = organizationDao.get().getByUserId(user.getId()) != null;
```

Comment:
Ore uses an `Organization` global role to keep track of org users and sync that status to auth; should we do the same?

Reply:
Yep, that's a better way of doing it, 1 less query.